docs/Contributing/API-for-contributors.md

```diff
@@ -3184,6 +3185,73 @@ Notifies the server about an agent error, resulting in two outcomes:
 
 `Status: 500`
 
+### Register a device with the setup experience
+
+`POST /api/v1/fleet/device/{token}/setup_experience/start`
+
+##### Parameters
+
+| Name  | Type   | In   | Description                        |
+| ----- | ------ | ---- | ---------------------------------- |
+| token | string | path | The device's authentication token. |
+
+##### Example
+
+`POST /api/v1/fleet/device/8b49859b-1ffa-483d-ad27-85b30aa3c55f/setup_experience/start`
+
+##### Default response
+
+`Status: 201`
+
+### Get the status of a device in the setup experience
+
+`GET /api/v1/fleet/device/{token}/setup_experience/status`
+
+##### Parameters
+
+| Name  | Type   | In   | Description                        |
+| ----- | ------ | ---- | ---------------------------------- |
+| token | string | path | The device's authentication token. |
+
+##### Example
+
+`POST /api/v1/fleet/device/8b49859b-1ffa-483d-ad27-85b30aa3c55f/setup_experience/status`
+
+##### Default response
+
+`Status: 200`
+
+```json
+{
+  "bootstrap_package": {
```

Comment:
@noahtalerman @georgekarrv we (@dantecatalfamo, @roperzh, and I) were chatting today about implementation details, and we saw that the bootstrap package is already installed as part of the "regular" (meaning before this feature) ADE enrollment flow that Fleet does. We believe we should be able to remove it from this endpoint because of that

Reply:
FYI @marko-lisica (giving you a ping here just in case you missed this)